src/ontology/reports/sync-synonym/doid.synonyms.confirmed.robot.tsv

```diff
@@ -1229,7 +1229,7 @@ MONDO:0001700	megaloblastic anemia	oio:hasExactSynonym		MGA1 Norwegian type			DO
 MONDO:0001700	megaloblastic anemia	oio:hasExactSynonym		megaloblastic anemia			DOID:13382	megaloblastic anemia					megaloblastic anemia	DOID:13382										
 MONDO:0001700	megaloblastic anemia	oio:hasExactSynonym		recessive hereditary megaloblastic anemia 1			DOID:13382	megaloblastic anemia					recessive hereditary megaloblastic anemia 1	DOID:13382										
 MONDO:0001700	megaloblastic anemia	oio:hasExactSynonym		recessive hereditary megaloblastic anaemia 1			DOID:13382	megaloblastic anemia		http://purl.obolibrary.org/obo/OMO_0003005			recessive hereditary megaloblastic anaemia 1	DOID:13382										
-MONDO:0001700	megaloblastic anemia	oio:hasExactSynonym		RH-MGA1			DOID:13382	megaloblastic anemia	http://purl.obolibrary.org/obo/mondo#ABBREVIATION				RH-MGA1	DOID:13382	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
+MONDO:0001700	megaloblastic anemia	oio:hasExactSynonym		RH-MGA1			DOID:13382	megaloblastic anemia	http://purl.obolibrary.org/obo/mondo#ABBREVIATION	http://purl.obolibrary.org/obo/mondo#ABBREVIATION			RH-MGA1	DOID:13382	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
```

Comment:
@joeflack4 can you confirm the formatting of this row looks correct? Thanks!

Reply:
In my local `mondo-edit`:
```
id: MONDO:0001700
name: megaloblastic anemia
synonym: "RH-MGA1" EXACT ABBREVIATION [DOID:13382]
```

The diff is caused by the recent addition of `ABBREVIATION`. In my local `mondo-edit`, it wasn't there until I did a `git pull`. So now that it is in `mondo-edit`, it shows up in the 
`synonym_type_mondo` column instead of just the `synonym_type` and `exact_synonym_type` columns.

Reply:
Is the formatting correct?

Reply:
I'm not 100% sure what you mean specifically by formatting, but yes, I opened the file, looked at all of the columns and their values and that row, and it looks correct.